src/components/planning/Planning.vue

```diff
@@ -189,7 +175,11 @@ export default {
       return this.mainUser.role.name === PLANNING_REFERENT;
     },
     personsGroupedBySector () {
-      return this.$_.groupBy(this.persons, 'sector._id');
+      if (this.isCustomerPlanning) {
+        return { allSector: this.persons };
+      } else {
+        return this.$_.groupBy(this.persons, 'sector._id');
+      }
```

Comment:
tu peux mettre `return this.isCustomerPlanning ? { allSector: this.persons } : this.$_.groupBy(this.persons, 'sector._id');`